README.md

```diff
@@ -117,35 +117,35 @@ For additional help, join us in the [Storybook Discord](https://discord.gg/story
-| [storysource](addons/storysource/)          | View the code of your stories within the Storybook UI                      |
-| [viewport](addons/viewport/)                | Change display sizes and layouts for responsive components using Storybook |
+| Addons                                                                    |                                                                            |
+|---------------------------------------------------------------------------|----------------------------------------------------------------------------|
+| [a11y](addons/a11y/)                                                      | Test components for user accessibility in Storybook                        |
+| [actions](addons/actions/)                                                | Log actions as users interact with components in the Storybook UI          |
+| [backgrounds](addons/backgrounds/)                                        | Let users choose backgrounds in the Storybook UI                           |
+| [cssresources](https://github.com/storybookjs/addon-cssresources)         | Dynamically add/remove css resources to the component iframe               |
+| [design assets](https://github.com/storybookjs/addon-design-assets)       | View images, videos, weblinks alongside your story                         |
+| [docs](addons/docs/)                                                      | Add high quality documentation to your components                          |
+| [events](https://github.com/storybookjs/addon-events)                     | Interactively fire events to components that respond to EventEmitter       |
+| [google-analytics](https://github.com/storybookjs/addon-google-analytics) | Reports google analytics on stories                                        |
+| [graphql](https://github.com/storybookjs/addon-graphql)                   | Query a GraphQL server within Storybook stories                            |
+| [jest](addons/jest/)                                                      | View the results of components' unit tests in Storybook                    |
+| [links](addons/links/)                                                    | Create links between stories                                               |
+| [query params](https://github.com/storybookjs/addon-queryparams)          | Mock query params                                                          |
+| [storyshots](addons/storyshots/)                                          | Snapshot testing for components in Storybook                               |
+| [storysource](addons/storysource/)                                        | View the code of your stories within the Storybook UI                      |
+| [viewport](addons/viewport/)                                              | Change display sizes and layouts for responsive components using Storybook |
```

Comment:
@jonniebigodes should we add [measure](https://github.com/storybookjs/addon-measure) and [outline](https://github.com/chromaui/storybook-outline) to this list too?

Reply:
We can, I refrained from doing it as they'll be released with 6.3 and not yet released. But I can update the pr to include them, no problem.

Reply:
Good point, no worries